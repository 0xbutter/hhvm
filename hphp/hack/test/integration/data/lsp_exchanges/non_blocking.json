[
  {
    "jsonrpc": "2.0",
    "id": 1,
    "method": "initialize",
    "params": {
      "initializeOptions": {},
      "processId": null,
      "rootPath": "${root_path}",
      "capabilities": {}
    }
  },
  {
    "jsonrpc": "2.0",
    "method": "textDocument/definition",
    "id": 2,
    "params": {
      "textDocument": {
        "uri": "${php_file_uri}"
      },
      "position": {
        "line": 7,
        "character": 11
      }
    }
  }
]
