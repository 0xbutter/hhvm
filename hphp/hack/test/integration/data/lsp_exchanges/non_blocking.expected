[
  {
    "jsonrpc": "2.0",
    "id": 1,
    "method": "window/showMessageRequest",
    "params": {
      "type": 3,
      "message": "Waiting for hh_server to be ready...",
      "actions": []
    }
  },
  {
    "jsonrpc": "2.0",
    "method": "telemetry/event",
    "params": {
      "type": 4,
      "message": "Version in hhconfig=[NoVersion]"
    }
  }
]
